src/main/scala/com/softwaremill/realworld/GlobalLoggingDefectHandler.scala

```diff
@@ -0,0 +1,20 @@
+package com.softwaremill.realworld
+
+import utils.Exceptions
+import sttp.model.StatusCode
+import sttp.monad.MonadError
+import sttp.tapir.server.interceptor.exception.{ExceptionContext, ExceptionHandler}
+import sttp.tapir.server.model.ValuedEndpointOutput
+import sttp.tapir.{statusCode, stringBody}
+import zio.{Cause, ZIO}
+
+class GlobalLoggingDefectHandler[F[_]] extends ExceptionHandler[F]:
+  override def apply(ctx: ExceptionContext)(implicit monad: MonadError[F]): F[Option[ValuedEndpointOutput[_]]] =
+    monad.unit({
+      ZIO.logErrorCause(Cause.fail(ctx.e))
```

Comment:
that's a no-op - an effect is created, but isn't sequenced and hence is never run